postprocessing/janus-pp-rec.c

```diff
@@ -1053,7 +1095,7 @@ int main(int argc, char *argv[])
 	while(tmp) {
 		count++;
 		if(!data)
-			JANUS_LOG(LOG_VERB, "[%10lu][%4d] seq=%"SCNu16", ts=%"SCNu64", time=%.2fs pts=%.2fs\n", tmp->offset, tmp->len, tmp->seq, tmp->ts, (double)(tmp->ts-list->ts)/(double)rate, (double)tmp->p_ts/1000);
+			JANUS_LOG(LOG_VERB, "[%10lu][%4d] seq=%"SCNu16", ts=%"SCNu64", time=%.2fs pts=%.2fs diff=%.2fs\n", tmp->offset, tmp->len, tmp->seq, tmp->ts, (double)(tmp->ts-list->ts)/(double)rate, (double)tmp->p_ts/1000, (double)(((double)tmp->p_ts/1000) - ((double)(tmp->ts-list->ts)/(double)rate)));
```

Comment:
I don't think this is strictly necessary, since the two needed data are already being printed out

Reply:
It's not necessary. I'll remove this change.